Frameworks/AddressBook/ABRecord.mm

```diff
@@ -65,10 +206,13 @@ bool ABRecordRemoveValue(ABRecordRef record, ABPropertyID property, CFErrorRef*
 }
 
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 CFStringRef ABRecordCopyCompositeName(ABRecordRef record) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    // Behavior on the reference platform is undefined for Source records,
+    // and Windows Contacts doesn't support Group records, so we can assume
+    // it is a Person record.
```

Comment:
do you want to add the fact that we are treating this as Person record in the notes?